tests/integration_tests/test_dependencies.py

```diff
@@ -42,8 +43,15 @@
         [BATCH_SIZE, SEQ_SIZE, HIDDEN_SIZE]
     ]
 )
-def test_multiple_dependencies(hidden_shape, dependent_hidden_shape,
-                               dependent_hidden_shape2):
+@pytest.mark.parametrize(
+    'reduce_dependencies', ['sum', 'mean', 'avg', 'max', 'concat', 'last']
```

Comment:
why not attention?

Reply:
Only reason I had those values was what I read in the Ludwig documentation:
```
reduce_dependencies (default sum): defines how to reduce the output of a dependent feature that is not a vector, but a matrix or a higher order tensor, on the first dimension 9second if you count the batch dimension). Available values are: sum, mean or avg, max, concat (concatenates along the first dimension), last (returns the last vector of the first dimension).
```
I can add `attention`.  I'll add this to the running list I'm maintaining re: updating the ludwig docs with this new value

Reply:
added 'attention` and resolved errors that were unique to the `attention` reducer